scripts/update-cdn-urls.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+/**
+ * @fileoverview
+ * Updates the CDN URLs in the README.md to the latest major version. Does
+ * nothing if the README.md is already up-to-date.
+ *
+ * It assumes the major version in the README is either equal to the current
+ * package major version, or the package minor version minus 1. If not the
+ * behaviour of this script should be consider undefined.
```

Comment:
I don't understand the statement "If not the behaviour of this script should be consider undefined.", seems confuse to me. Really, if the already defined major version in the README is not the one lower than the major version defined in the `package.json` file, the script does nothing, except for the case that the version defined in the README would be a number with different number of decimals, for example:

- `package.json`: "6.2.0"
- `README`: `v55.2.0`
- `README` after script: `v65.2.0`

It is a stupid case, but for consistency I would suggest to remove that statement and applies the other suggestion that I've mentioned below.

```suggestion
 * package major version, or the package minor version minus 1.
```

Reply:
> I don't understand the statement "If not the behaviour of this script should be consider undefined.", seems confuse to me. Really, if the already defined major version in the README is not the one lower than the major version defined in the `package.json` file, the script does nothing, except for the edge case that the version defined in the README would be defined with different number of decimals, [...]

I believe the statement is self-explanatory, if you consider _why_ I wrote it. Of course, I _could_ describe the behaviour of the script in the scenario the input conditions are not met, but why would I? The input conditions should be met, and if they're not, something has already gone wrong. Moreover, if the behaviour in the invalid case changes, now I need to update the documentation for something that is irrelevant...  Why would I do that.

You can really read it just like this: "If you don't meet my preconditions, I cannot give any guarantees on what is going to happen. Of course something is going to happen, but I'm not going to worry about it." You can well argue that we should prevent that, maybe we should, or maybe we can fix the undefined behaviour. But when I wrote this I didn't feel like that's worth it: most of the time it's going to work as expected and when it doesn't it is easy to fix - so it's not worth the effort to solve the problem.

Also, of the two scenarios, consider which one is more likely? It must be the scenario that the version in the README is not exactly one less than the version in `package.json`, as the other is a subset of that. So which one should we fix? Probably the former.

So I did that, turns out it didn't take that much effort but when I put this Pull Request together I didn't think about it 🙃